tests/common-lib/clusterAccess.js

```diff
@@ -6,31 +6,72 @@ const { execSync } = require('child_process')
 const request = require('supertest')
 const fs = require('fs')
 
-// Login to the cluster
-const clusterLogin = () => {
-  if (!process.env.USE_HUB_KUBECONFIG) {
-    execSync(
-      `oc login -u ${config.get('options:hub:user')} -p ${config.get(
-        'options:hub:password'
-      )} --server=https://api.${config.get(
-        'options:hub:baseDomain'
-      )}:6443 --insecure-skip-tls-verify`
-    )
-  } else {
-    execSync(
-      `oc config use-context --kubeconfig=${process.env.OPTIONS_HUB_KUBECONFIG} ${process.env.OPTIONS_HUB_KUBECONTEXT}`
-    )
+/**
+ * Login into the cluster environment with the `oc` cli command.
+ * @param {object} options Additional options for logging into the cluster environment.
+ */
+const clusterLogin = (options = { useInsecure: true }) => {
+  var cmd = `oc login -u ${config.get('options:hub:user')} -p ${config.get(
+    'options:hub:password'
+  )} --server=https://api.${config.get('options:hub:baseDomain')}:6443`
+
+  if (options.useInsecure) {
+    cmd += ` --insecure-skip-tls-verify`
   }
+
+  execSync(cmd)
 }
 
-// Login and get access token
-const getToken = () => {
-  clusterLogin()
-  return execSync('oc whoami -t').toString().replace('\n', '')
+/**
+ * Delete a pod resourcec from a specified namespace within the cluster environment.
+ * @param {*} pod The pod of the resource object.
+ * @param {object} options Additional options for deleting the pod resource from the cluster environment.
+ * @param {*} ns Teh namespace of the pod resource object.
+ */
+async function deletePod(pod, ns, options = {}) {
+  execSync(`oc delete pod ${pod} -n ${ns} --wait=true`)
+}
```

Comment:
You could make this function more generic:
```js
async function deleteResource(kind, name, ns, options = {}) {
    execSync(`oc delete ${kind} ${name} -n ${ns} --wait=true`)
}
```

Reply:
Making the function more generic.